server/neptune/workflows/internal/deploy/workflow.go

```diff
@@ -75,8 +76,11 @@ func newRunner(ctx workflow.Context, request Request, tfWorkflow terraform.Workf
 	revisionReceiver := revision.NewReceiver(ctx, revisionQueue, a, sideeffect.GenerateUUID)
 	tfWorkflowRunner := terraform.NewWorkflowRunner(a, tfWorkflow)
 	revisionProcessor := &queue.RevisionProcessor{
-		Activities:              a,
-		TerraformWorkflowRunner: tfWorkflowRunner,
+		Activities: a,
+		TerraformWorkflowRunner: &lyft.AuditWorkflowRunnerWrapper{
```

Comment:
this is fine for now, but I think in the future, we will want to make our own specific features less invasive to the codebase if this is an oss project for the future

Reply:
Do you mean setting up the workflow runner similar to how we setup the [project_runners](https://github.com/lyft/atlantis/blob/8251f822888a69ae5be9853da104e6423ce81e2c/server/wrappers/project_runners.go#L47)? I initially thought of it but since we're only adding auditing support for now, I figured it'd be an overkill so instead, I decided to initialize it directly in the workflow code.

Reply:
I mean create a config that determines whether or not the workflow makes a decision to signal another workflow this metadata. We can define that workflow internally and have it perform this activity.

Reply:
Ahh right, we can tackle that as an improvement!